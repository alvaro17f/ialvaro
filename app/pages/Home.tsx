import Wobble from "@/components/Wobble";

export default function Home() {
	return (
		<section
			id="home"
			className="h-[100dvh] top-0 grid place-items-center gap-5"
		>
			<div>
				<Wobble sentence="ALVARO" style="hover:text-azama-primary" />
				<Wobble sentence="GARCIA" style="hover:text-azama-danger" />
				<Wobble sentence="MACIAS" style="hover:text-azama-primary" />
			</div>
		</section>
	);
}
