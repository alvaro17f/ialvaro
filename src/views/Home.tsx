import Wobble from "src/components/Wobble";

export default function Home() {
	return (
		<section
			id="home"
			className="h-[100dvh] top-0 grid place-items-center gap-5"
		>
			<div>
				<Wobble sentence="ALVARO" style="hover:text-alvaro-primary" />
				<Wobble sentence="GARCIA" style="hover:text-alvaro-danger" />
				<Wobble sentence="MACIAS" style="hover:text-alvaro-primary" />
			</div>
		</section>
	);
}
