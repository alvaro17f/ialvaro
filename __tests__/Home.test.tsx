import { expect, it, describe, beforeEach, afterEach } from "vitest";
import {
	render,
	screen,
	within,
	fireEvent,
	cleanup,
} from "@testing-library/react";
import Home, { metadata } from "@/app/page";

describe("<Home />", () => {
	beforeEach(() => {
		render(<Home />);
	});
	afterEach(cleanup);

	it("should match the snapshot", () => {
		const { container } = render(<Home />);
		expect(container).toMatchSnapshot();
	});

	it("should render the <Header />", () => {
		expect(
			screen.getByRole("heading", { level: 1, name: /home/i }),
		).toBeDefined();
	});

	it("should match metadata's title to <Header /> title", () => {
		expect(metadata.title).toStrictEqual("Home");
	});

	it("should render the <Content />", () => {
		expect(screen.getAllByLabelText(/content/i).length).toBe(2);
	});
});
