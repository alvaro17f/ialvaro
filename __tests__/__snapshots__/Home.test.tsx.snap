// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Home /> > should match the snapshot 1`] = `
<div>
  <title>
    Home - [ iAlvaro ]
  </title>
  <section
    class="h-[100dvh] top-0 grid place-items-center gap-5"
    id="home"
  >
    <div>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        A
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        L
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        V
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        A
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        R
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        O
      </span>
      <br />
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-danger"
      >
        G
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-danger"
      >
        A
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-danger"
      >
        R
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-danger"
      >
        C
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-danger"
      >
        I
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-danger"
      >
        A
      </span>
      <br />
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        M
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        A
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        C
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        I
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        A
      </span>
      <span
        class="inline-block cursor-default text-6xl md:text-9xl hover:text-alvaro-primary"
      >
        S
      </span>
      <br />
    </div>
  </section>
  <section
    id="biography"
  >
    <section
      aria-label="content"
      class="p-5 mb-5 rounded-lg text-alvaro-white"
      style="opacity: 0;"
    >
      <article
        class="p-5"
      >
        <div
          class="grid md:gap-10 md:grid-cols-[1fr_2fr] place-items-center h-[100dvh] mb-32 md:mb-0"
        >
          <div
            class="mb-5"
          >
            <a
              href="https://linkedin.com/in/alvarogarciamacias"
              rel="noreferrer"
              target="_blank"
            >
              <img
                alt="profile picture"
                class="border-[7px] rounded-full border-alvaro-white hover:border-dashed cursor-pointer"
                height="200"
                src="/images/profile.png"
                width="200"
              />
            </a>
          </div>
          <div
            class="max-w-[75ch]"
          >
            <h2
              class="mb-4 text-4xl"
            >
              Hello 
              <span
                class="text-alvaro-danger"
              >
                World
              </span>
              !
            </h2>
            <p>
              My name is
               
              <span
                class="text-xl text-alvaro-primary"
              >
                √Ålvaro
              </span>
               and I am a Full Stack Developer.
              <br />
              I feel very fortunate to have found what I am passionate about and to make it my job.
              <br />
              I have also worked managing, training and leading marketing teams at a national level.
              <br />
              If there is one thing that characterises me, it is that I am 100% involved in everything I do.
              <br />
              I am hard-working, perfectionist and non-conformist.
              <br />
              I have a direct and creative working style. And I put all the energy I get from working in what I am so enthusiastic about.
            </p>
          </div>
        </div>
      </article>
    </section>
  </section>
  <section
    id="skills"
  >
    <div
      class="h-24"
    />
    <section
      class="p-2 mb-5 rounded-lg text-alvaro-dark from-alvaro-primary to-transparent bg-gradient-to-r"
      style="width: 0%; opacity: 0; transform: scaleY(0.5) translateZ(0);"
    >
      <h1
        class="text-5xl md:text-3xl"
        style="opacity: 0;"
      >
        Skills
      </h1>
    </section>
    <div
      class="grid grid-cols-2 gap-5 md:grid-cols-4 place-items-center"
    >
      <div
        class="grid place-content-center"
      >
        <a
          href="https://react.dev/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="react.js"
              class="w-40 h-40 p-3"
              src="/skills/react.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              react.js
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://vue.org/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="vue.js"
              class="w-40 h-40 p-3"
              src="/skills/vue.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              vue.js
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://typescriptlang.org/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="typescript"
              class="w-40 h-40 p-3"
              src="/skills/typescript.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              typescript
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://javascript.com/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="javascript"
              class="w-40 h-40 p-3"
              src="/skills/javascript.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              javascript
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://tailwindcss.com/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="tailwind"
              class="w-40 h-40 p-3"
              src="/skills/tailwind.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              tailwind
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://nextjs.org/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="next.js"
              class="w-40 h-40 p-3"
              src="/skills/next.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              next.js
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://rust-lang.org/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="rust"
              class="w-40 h-40 p-3"
              src="/skills/ferris.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              rust
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://go.dev/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="go"
              class="w-40 h-40 p-3"
              src="/skills/go.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              go
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://jestjs.io/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="jest"
              class="w-40 h-40 p-3"
              src="/skills/jest.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              jest
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://nodejs.org/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="node.js"
              class="w-40 h-40 p-3"
              src="/skills/node.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              node.js
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://docker.com/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="docker"
              class="w-40 h-40 p-3"
              src="/skills/docker.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              docker
            </h1>
          </div>
        </a>
      </div>
      <div
        class="grid place-content-center"
      >
        <a
          href="https://mysql.com/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="grid gap-2 p-5 text-center border-dashed cursor-pointer place-items-center rounded-xl"
            style="opacity: 0; transform: none;"
            tabindex="0"
          >
            <img
              alt="sql"
              class="w-40 h-40 p-3"
              src="/skills/sql.svg"
            />
            <h1
              class="mb-5 text-xl text-alvaro-white"
            >
              sql
            </h1>
          </div>
        </a>
      </div>
    </div>
  </section>
  <section
    id="experience"
  >
    <div
      class="h-24"
    />
    <section
      class="p-2 mb-5 rounded-lg text-alvaro-dark from-alvaro-primary to-transparent bg-gradient-to-r"
      style="width: 0%; opacity: 0; transform: scaleY(0.5) translateZ(0);"
    >
      <h1
        class="text-5xl md:text-3xl"
        style="opacity: 0;"
      >
        Experience
      </h1>
    </section>
    <div
      class="grid md:grid-cols-[1fr_2fr] mt-12 md:gap-10"
    >
      <div
        class="grid gap-2 border-dashed cursor-pointer border-[4px] border-alvaro-muted b-20 md:p-5 h-60 md:mb-36 place-items-center rounded-xl"
        style="opacity: 0; transform: none;"
        tabindex="0"
      >
        <a
          href="https://devoteam.com/"
          target="_blank"
        >
          <img
            alt="devoteam"
            class="w-56 h-36"
            src="/experience/devoteam.svg"
          />
        </a>
        <p>
          October 2021
           - 
          April 2023
        </p>
      </div>
      <div
        class="mt-12 md:mt-0"
      >
        <h1
          class="mb-5 text-4xl md:text-6xl text-alvaro-primary"
          style="opacity: 0;"
        >
          devoteam
        </h1>
        <div
          class="mb-20 whitespace-pre-wrap"
          style="opacity: 0;"
        >
          <p
            class="text-xl underline text-alvaro-danger"
          >
            Client
          </p>
          RSI
          <br />
          <br />
          <p
            class="text-xl underline text-alvaro-danger"
          >
            Projects
          </p>
          Frontend management for Banco Caminos, Arquia, Vall Banc, Fiare, Bancofar, etc...
          <br />
          <br />
          <p
            class="text-xl underline text-alvaro-danger"
          >
            Tasks
          </p>
          - Web application development using Vue.js and React.js
- REST API integration in front-end applications
- Unit and integration test development
- Code documentation
- Translations using i18n
- Modification and elaboration of login pages
- Internal SSO access elaboration
- Modification of CSS styles
          <br />
          <br />
          <p
            class="text-xl underline text-alvaro-danger"
          >
            Skills
          </p>
          vue.js
react.js
typescript
javascript
jest.js
node.js
html
css
jsp
oracle db
inkscape
jira
git
dbeaver
postman
invision
overflow
        </div>
      </div>
    </div>
    <div
      class="grid md:grid-cols-[1fr_2fr] mt-12 md:gap-10"
    >
      <div
        class="grid gap-2 border-dashed cursor-pointer border-[4px] border-alvaro-muted b-20 md:p-5 h-60 md:mb-36 place-items-center rounded-xl"
        style="opacity: 0; transform: none;"
        tabindex="0"
      >
        <a
          href=""
          target="_blank"
        >
          <img
            alt="your company"
            class="w-56 h-36"
            src="/experience/company.svg"
          />
        </a>
        <p>
          April 2023
           - 
          now
        </p>
      </div>
      <div
        class="mt-12 md:mt-0"
      >
        <h1
          class="mb-5 text-4xl md:text-6xl text-alvaro-primary"
          style="opacity: 0;"
        >
          your company
        </h1>
        <div
          class="mb-20 whitespace-pre-wrap"
          style="opacity: 0;"
        >
          <p
            class="text-xl underline text-alvaro-danger"
          />
          <br />
          <br />
          <p
            class="text-xl underline text-alvaro-danger"
          />
          <br />
          <br />
          <p
            class="text-xl underline text-alvaro-danger"
          />
          <br />
          <br />
          <p
            class="text-xl underline text-alvaro-danger"
          />
        </div>
      </div>
    </div>
  </section>
  <section
    id="resume"
  >
    <div
      class="h-24"
    />
    <a
      href="/assets/resume.pdf"
      rel="noreferrer"
      target="_blank"
    >
      <div
        aria-label="content"
        class="grid place-items-center p-5 mb-5 rounded-xl border-[5px] border-alvaro-danger text-alvaro-white"
        style="opacity: 0;"
        tabindex="0"
      >
        <h1
          class="text-5xl text-center"
        >
          GET MY RESUME FOR FREE!
        </h1>
        <img
          alt="download"
          class="h-20"
          src="/skills/react.svg"
        />
      </div>
    </a>
  </section>
  <section
    class="h-[100dvh]"
    id="contact"
  >
    <div
      class="h-24 md:h-36"
    />
    <section
      aria-label="section-form"
      class="grid grid-cols-1 md:grid-cols-[1fr_2fr] place-items-center p-5 mb-5 rounded-xl text-alvaro-white border border-alvaro-muted"
      style="opacity: 0;"
    >
      <div
        class="mb-5 md:mb-0 md:text-center"
      >
        <h2
          class="mb-5 text-3xl"
        >
          CONTACT ME!
        </h2>
        <div
          class="grid grid-cols-2 place-items-center"
        >
          <a
            href="https://github.com/alvaro17f"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              class="fill-alvaro-white hover:fill-alvaro-primary"
              height="48px"
              viewBox="0 0 64 64"
              width="48px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M 32 10 C 19.85 10 10 19.85 10 32 C 10 44.15 19.85 54 32 54 C 44.15 54 54 44.15 54 32 C 54 19.85 44.15 10 32 10 z M 32 14 C 41.941 14 50 22.059 50 32 C 50 40.238706 44.458716 47.16934 36.904297 49.306641 C 36.811496 49.1154 36.747844 48.905917 36.753906 48.667969 C 36.784906 47.458969 36.753906 44.637563 36.753906 43.601562 C 36.753906 41.823563 35.628906 40.5625 35.628906 40.5625 C 35.628906 40.5625 44.453125 40.662094 44.453125 31.246094 C 44.453125 27.613094 42.554688 25.720703 42.554688 25.720703 C 42.554688 25.720703 43.551984 21.842266 42.208984 20.197266 C 40.703984 20.034266 38.008422 21.634812 36.857422 22.382812 C 36.857422 22.382813 35.034 21.634766 32 21.634766 C 28.966 21.634766 27.142578 22.382812 27.142578 22.382812 C 25.991578 21.634813 23.296016 20.035266 21.791016 20.197266 C 20.449016 21.842266 21.445312 25.720703 21.445312 25.720703 C 21.445312 25.720703 19.546875 27.611141 19.546875 31.244141 C 19.546875 40.660141 28.371094 40.5625 28.371094 40.5625 C 28.371094 40.5625 27.366329 41.706312 27.265625 43.345703 C 26.675939 43.553637 25.872132 43.798828 25.105469 43.798828 C 23.255469 43.798828 21.849984 42.001922 21.333984 41.169922 C 20.825984 40.348922 19.7845 39.660156 18.8125 39.660156 C 18.1725 39.660156 17.859375 39.981656 17.859375 40.347656 C 17.859375 40.713656 18.757609 40.968484 19.349609 41.646484 C 20.597609 43.076484 20.574484 46.292969 25.021484 46.292969 C 25.547281 46.292969 26.492043 46.171872 27.246094 46.068359 C 27.241926 47.077908 27.230199 48.046135 27.246094 48.666016 C 27.251958 48.904708 27.187126 49.114952 27.09375 49.306641 C 19.540258 47.168741 14 40.238046 14 32 C 14 22.059 22.059 14 32 14 z"
              />
            </svg>
          </a>
          <a
            href="https://linkedin.com/in/alvarogarciamacias"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              class="fill-alvaro-white hover:fill-alvaro-primary"
              height="48px"
              viewBox="0 0 64 64"
              width="48px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M 23.773438 12 C 12.855437 12 12 12.854437 12 23.773438 L 12 40.226562 C 12 51.144563 12.855438 52 23.773438 52 L 40.226562 52 C 51.144563 52 52 51.145563 52 40.226562 L 52 23.773438 C 52 12.854437 51.145563 12 40.226562 12 L 23.773438 12 z M 21.167969 16 L 42.832031 16 C 47.625031 16 48 16.374969 48 21.167969 L 48 42.832031 C 48 47.625031 47.624031 48 42.832031 48 L 21.167969 48 C 16.374969 48 16 47.624031 16 42.832031 L 16 21.167969 C 16 16.374969 16.374969 16 21.167969 16 z M 22.501953 18.503906 C 20.872953 18.503906 19.552734 19.824172 19.552734 21.451172 C 19.552734 23.078172 20.871953 24.400391 22.501953 24.400391 C 24.126953 24.400391 25.447266 23.079172 25.447266 21.451172 C 25.447266 19.826172 24.126953 18.503906 22.501953 18.503906 z M 37.933594 26.322266 C 35.473594 26.322266 33.823437 27.672172 33.148438 28.951172 L 33.078125 28.951172 L 33.078125 26.728516 L 28.228516 26.728516 L 28.228516 43 L 33.28125 43 L 33.28125 34.949219 C 33.28125 32.826219 33.687359 30.771484 36.318359 30.771484 C 38.912359 30.771484 38.945312 33.200891 38.945312 35.087891 L 38.945312 43 L 44 43 L 44 34.074219 C 44 29.692219 43.054594 26.322266 37.933594 26.322266 z M 19.972656 26.728516 L 19.972656 43 L 25.029297 43 L 25.029297 26.728516 L 19.972656 26.728516 z"
              />
            </svg>
          </a>
        </div>
      </div>
      <form
        aria-label="contact-form"
        class="grid gap-5 p-2 place-items-center md:w-full"
        name="<input type=\\"text\\" name=\\"name\\" placeholder=\\"Name *\\" aria-label=\\"name\\" class=\\"bg-alvaro-base placeholder:text-alvaro-danger text-alvaro-white max-w-[15rem] md:w-[42rem] md:max-w-none p-5  border focus:border-alvaro-danger border-alvaro-muted rounded-xl focus:outline-none\\" required=\\"\\" value=\\"\\">"
      >
        <input
          aria-label="name"
          class="bg-alvaro-base placeholder:text-alvaro-danger text-alvaro-white max-w-[15rem] md:w-[42rem] md:max-w-none p-5  border focus:border-alvaro-danger border-alvaro-muted rounded-xl focus:outline-none"
          name="name"
          placeholder="Name *"
          required=""
          type="text"
          value=""
        />
        <input
          aria-label="email"
          class="bg-alvaro-base placeholder:text-alvaro-danger text-alvaro-white max-w-[15rem] md:w-[42rem] md:max-w-none p-5  border border-alvaro-muted rounded-xl focus:outline-none focus:border-alvaro-danger"
          name="email"
          placeholder="Email *"
          required=""
          type="email"
          value=""
        />
        <input
          aria-label="phone"
          class="bg-alvaro-base placeholder:text-alvaro-muted text-alvaro-white max-w-[15rem] md:w-[42rem] md:max-w-none p-5 border border-alvaro-muted rounded-xl focus:border-alvaro-primary focus:outline-none"
          name="phone"
          placeholder="Phone"
          type="text"
          value=""
        />
        <textarea
          aria-label="message"
          class="bg-alvaro-base text-alvaro-white max-w-[15rem] md:w-[42rem] md:max-w-none p-8 border border-alvaro-muted focus:border-alvaro-danger rounded-xl placeholder:text-alvaro-danger focus:outline-none"
          name="message"
          placeholder="Your message goes here... *"
          required=""
        />
        <button
          aria-label="submit"
          class="p-0 m-5 text-5xl rounded-full shadow-md md:w-40 shadow-alvaro-base bg-alvaro-base"
          tabindex="0"
          type="submit"
        >
          üî•
        </button>
      </form>
    </section>
    <div
      class="h-28"
    />
    <div
      style="position: fixed; z-index: 9999; top: 16px; left: 16px; right: 16px; bottom: 16px; pointer-events: none;"
    />
  </section>
</div>
`;
